index.html

```diff
@@ -4127,7 +4127,7 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
                <tr>
                  <th>Actual value</th>
                  <th>Stored Decimal values</th>
-                 <th>Stored 2-byte Hex unsigned values</th>
+                 <th>Stored 4-byte Hex unsigned values</th>
```

Comment:
Shouldn't this simply say "4 bytes" and then, in the prose, 

> Each value is encoded as a [PNG four-byte unsigned integer](https://w3c.github.io/PNG-spec/#dfn-png-four-byte-unsigned-integer)

Reply:
I'm updating this PR right now to land it.
I was about to comment "Should this mention network byte order? That gets wordy. Maybe link to the PNG four-byte thing?" Then I saw your comment.

But I think you may have meant something different by 'prose'. You don't mean the table header text here should link, right? Rather, before the tables there should be some wording to explain that it is the PNG four-byte and link there?

Reply:
I fixed the rest and looped back to this. I was going to add your suggested text just before the examples. But I notice right there it already explains which values are PNG two-byte unsigned integers and which are PNG four-byte unsigned integers.

I think we can leave that existing text.